src/scripts/main.js

```diff
@@ -18,10 +18,27 @@ const food = {
   },
 };
 
+
+console.log(``)
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const item in data) {
+    const li = document.createElement('li');
+
+    li.textContent = item;
+
+    if (Object.keys(item).length > 0) {
```

Comment:
The condition `if (Object.keys(item).length > 0)` is incorrect. `item` is a string (the key of the object), not an object itself. You should check `if (Object.keys(data[item]).length > 0)` to determine if the current item has children.